packages/blocks/src/lib/navigation-map/components/center-inputs.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang='ts'>
+
+import { useMapLibre, type LngLat } from '$lib';
+import LngLatInput from '../components/input/lnglat.svelte';
+
+const { map, mapCenter } = useMapLibre();
+
+const handleInput = (event: CustomEvent<LngLat>) => {
+  map.jumpTo({ center: event.detail })
+}
+
+</script>
+
+<div class='absolute right-10 flex flex-wrap gap-y-2 items-end justify-between py-2.5 w-60 px-4 z-[1000]'>
```

Comment:
Should `z-[1000]` be added to the tailwind config as something more descriptive? `z-max` or similar so it's intent is a little more clear?

Reply:
Agreed, added to theme